### PR TITLE
Add Issue Template for Minecraft

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

**Comment by KalleOlaviNiemitalo** [comment]
Alternatively, you could define `contact_links` in `.github/ISSUE_TEMPLATE/config.yml` as in <https://github.com/microsoft/terminal/pull/8114>. It's documented in [Configuring the template chooser](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

**Comment by karianna** [comment]
> Alternatively, you could define `contact_links` in `.github/ISSUE_TEMPLATE/config.yml` as in [microsoft/terminal#8114](https://github.com/microsoft/terminal/pull/8114). It's documented in [Configuring the template chooser](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

Thank you!  PR Updated.

**Comment by KalleOlaviNiemitalo** [comment]
> Please submit Minecraft related issues to the Mojang folks will take the first pass at triaging the issue.

Missing words?

> we'll work together to reolve it

Missing letter.

**Comment by karianna** [comment]
> > Please submit Minecraft related issues to the Mojang folks will take the first pass at triaging the issue.
> 
> Missing words?
> 
> > we'll work together to reolve it
> 
> Missing letter.

Ugh, the Keyboard and I are not friends today, updated in place :-)

**Comment by KalleOlaviNiemitalo** [comment]
If I click "New issue" at <https://github.com/microsoft/openjdk/issues> or <https://github.com/microsoft/terminal/issues> with a Web browser that has not signed in to GitHub, then GitHub doesn't show the issue templates and contact links; it instead asks me to sign up. To give a nicer experience to Minecraft players who do not already have GitHub accounts, perhaps it would be good to change the JRE itself to mention where to report Minecraft problems. That is not a reason to reject this pull request, though.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -24,15 +25,17 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+
+- OS: [e.g. iOS]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
```
Comment (d3r3kk) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Maybe consider being more explicit here?
```suggestion

- OS & version [e.g. Windows 10, 1822]:
- Browser & version [e.g. Edge, 91.0.864.48]:
```

**Review by d3r3kk — COMMENTED** [review]
LGTM, with a modest suggestion.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+blank_issues_enabled: true
+
+contact_links:
+  - name: Minecraft Java Edition Related Bug 🔐
+    url: https://bugs.mojang.com/projects/MC/issues/
+    about: We love Minecraft and would love to help! However, this probably isn't quite the right place to report this issue :-). Please submit Minecraft Java Edition related issues to the Mojang folks who will take the first pass at triaging the issue. If it is a genuine Microsoft Build of OpenJDK (Java/JVM) issue then they will open an issue with us, and we'll work together to resolve it. For context, most Java crash reports relating to Minecraft Java Edition are the result of Java calling a buggy native library, O/S driver etc. So although you'll see a _Java_ crash report, the root cause is typically not Java itself, but the inside the library that Java called.
```
Comment (brstaffoMS) [inline .github/ISSUE_TEMPLATE/config.yml:6]: Typo: "but the inside the"

**Comment by karianna** [comment]
> If I click "New issue" at https://github.com/microsoft/openjdk/issues or https://github.com/microsoft/terminal/issues with a Web browser that has not signed in to GitHub, then GitHub doesn't show the issue templates and contact links; it instead asks me to sign up. To give a nicer experience to Minecraft players who do not already have GitHub accounts, perhaps it would be good to change the JRE itself to mention where to report Minecraft problems. That is not a reason to reject this pull request, though.

Thanks for that suggestion, we'll mull over that for the next quarterly release there are some pros and cons (mostly around the maintenance of the one special change) that we need to think about :-).

**Pull request merged**